models/lighthouse.js

```diff
@@ -25,15 +27,24 @@
  *	 lighthouseInfo.scores {array}
  */
 class Lighthouse {
-  constructor(pwaId, absoluteStartUrl, lighthouseInfo) {
+  constructor(pwaId, absoluteStartUrl, lighthouseJson) {
     this.pwaId = pwaId;
     this.absoluteStartUrl = absoluteStartUrl;
-    this.lighthouseInfo = lighthouseInfo;
-    this.totalScore = lighthouseInfo.totalScore;
-    this.lighthouseVersion = lighthouseInfo.lighthouseVersion;
+    this.lighthouseInfo = libLighthouse.processLighthouseJson(lighthouseJson);
+    this._lighthouseJson = JSON.stringify(lighthouseJson);
```

Comment:
I can change the constructor to receive just the string and avoid parsing twice.